cmd/server/config.go

```diff
@@ -7,12 +7,12 @@ const (
 
 	// Server
 	defaultServerAddress = "0.0.0.0"
-	defaultServerPort    = "9090"
+	defaultServerPort    = "9099"
```

Comment:
Here and below: see corresponding comment in PortalInfoblox repo

Reply:
done